tools/postman/Makefile

```
@@ -1,6 +1,7 @@
 export OPENAPI_FOLDER=./openapi
 export TMP_FOLDER=./tmp
 export FULL_OPENAPI_FOLDER=../../openapi/v2/
+ENV_FILE = ./local.env
```

```suggestion
ENV_FILE = ./.env
```

---

Alternatively (if we want to export env vars):

```suggestion
ENV_FILE = ./env.sh
```